docs/src/testutils.md

```diff
@@ -1,7 +1,7 @@
 # TestUtils
 
-Provides test fakes, [`FakeTemplate`](@ref) and [`FakeModel`](@ref), that are useful for
-testing downstream dependencies, and [`test_interface`](@ref) for testing the Model's API
+Provides test fakes, `FakeTemplate` and `FakeModel`, that are useful for
+testing downstream dependencies, and `test_interface` for testing the model's API
```

Comment:
where'd the `@ref`s go?

Reply:
Ah I forgot that they work inside the docs just not the README. My bad.